.gitpod.yml.original

```diff
@@ -0,0 +1,11 @@
+image:
+  file: .gitpod.Dockerfile
+
+tasks:
+  - init: make release
+
+vscode:
+  extensions:
+    - timonwong.shellcheck@0.9.0:hsU/Rd39aqPYowTqL+DbXg==
+    - medo64.render-crlf@1.5.3:HLx6qIeeDMkxZSYLpmfjnw==
+    - exiasr.hadolint@0.3.0:/FkAW3eTb+z9O7VZAyAa0w==
```

Comment:
Let me know if you want hadolint in your workspace since it needs custom logic to get on ubuntu